boards/arm64/mimx8mm_evk_a53/mimx8mm_evk_a53.yaml

```diff
@@ -0,0 +1,12 @@
+identifier: mimx8mm_evk_a53
+name: NXP i.MX8M Mini EVK A53
+type: mcu
+arch: arm
```

Comment:
The failure you see on CI are caused by this. It should be `arm64`.

Reply:
oh. I missed to change this after aarch64 has a seperate folder.